.travis.yml

```diff
@@ -87,3 +87,18 @@ jobs:
         - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
             bash -x tools/lint-pr-commit-message.sh ${TRAVIS_PULL_REQUEST};
           fi
+
+    - name: "Python 3 is EXPERIMENTAL"
+      language: node_js
+      node_js: "node"
+      install:
+        - pyenv global 3.7.1
```

Comment:
Why not 3.7.4 since it's the latest.

Reply:
3.7.4 is available on the __pyenv__ on Travis CI.  __pyenv list --install__

We might even need to temporarily downgrade to Python 3.6 because __async__ is a reserved word in Python >= 3.7 and that messes with [__inspector_protocol/code_generator.py__](https://github.com/nodejs/node/blob/master/tools/inspector_protocol/code_generator.py#L112)